lib/hex_web/package.ex

```diff
@@ -157,6 +157,10 @@ defmodule HexWeb.Package do
     |> Enum.any?
   end
 
+  def last_owner?(package) do
+      (length owners package)  == 1
```

Comment:
Too much indentation and please use parenthesis around arguments instead of around the whole expression.